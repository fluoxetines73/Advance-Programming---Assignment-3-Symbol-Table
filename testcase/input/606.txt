PRINT
PRINT
INSERT f_FF string
INSERT g_1 number random
LOOKUP g_1
PRINT
ASSIGN f_FF 'Test123'