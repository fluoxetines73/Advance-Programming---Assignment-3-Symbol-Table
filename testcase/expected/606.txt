Invalid: INSERT g_1 number random